### ISSUE TITLE
MUFI entities

### ISSUE THREAD

**Issue by Crissov** [w3c/xml-entities#3]
I'm not sure whether it would be useful or practical, but it might be worth considering to add named character entities defined by the Medieval Unicode Font Initiative (MUFI). They are primarily intended for PUA characters (which is out of scope of this project), but most of them could be represented by a substitute sequence of standard Unicode characters, the ligatures in particular &ndash; alas, no such mapping seems to be available from MUFI itself. Furthermore, characters successfully proposed by MUFI for inclusion in Unicode still have named entities defined, which are likely unavailable in other sets and could therefore be in scope of this project. However, I do not see a simple single machine-readable file listing them all on the homepage.

- MUFI Homepage: [mufi.info](http://mufi.info)
  - new redirect target: <https://skaldic.abdn.ac.uk/m.php?p=mufi>
  - old redirect target: <https://folk.uib.no/hnooh/mufi/specs/>
- [MUFI character database](https://skaldic.abdn.ac.uk/db.php?if=mufi&table=mufi_char)
- [MUFI 4.0 release note](http://bora.uib.no/handle/1956/10699)
- [Wikipedia entry](https://en.wikipedia.org/wiki/Medieval_Unicode_Font_Initiative)

**Comment by davidcarlisle** [comment]
Since we started tracking these things in the 1990s we've only added new new name, so I fear this is out of scope sorry.

Adding the data to `unicode.xml`  might be in scope (just as it tracks tex and wolfram and elesvier etc names for characters) however I don't think there would be any consensus to adding more names to the html/xhtml/mathml entity set. 

Especially in the HTML parser incarnation of these names, there is a very high cost to changing the list in any way, there is no declarative DTD declaration that is read in, the list is specified in the code of each HTML parser.

As the list seems primarily aimed at characters not in Unicode (if we discount the PUA) I think just listing the overlap with declared Unicode characters in unicode.xml  might not be that useful (and make the file that much bigger to process for its intended uses)
